packages/web-styles/src/variables/_color.scss

```diff
@@ -117,10 +117,15 @@ $colors: map.merge(
 );
 $colors: map.merge($background-colors, $colors);
 
-// Compile a list of dark backgrounds, used in the :is selector mixin at mixins/color
+// Compile lists of backgrounds, used in the :is selector mixins at mixins/color
+$all-backgrounds: () !default;
```

Comment:
Do we still need the `$all-backgrounds` variable?

Reply:
Of course not! *shameonme*